docs/detections/alerts-view-details.asciidoc

```diff
@@ -26,7 +26,11 @@ The *Overview* tab contains these features:
 
 * *Summary*: Displays general details such as the alert's status, severity, risk score, and a link to the detection rule that produced the alert.
 
-* *Reason statement*: Provides a description of what generated the alert and provides general alert details. You can use this to understand the alert's origin and determine if the alert is relevant to your investigation.
+* *Alert-rendered reason statement*: Summarizes alert details and assembles the information into a human-readable format. High-level details are provided, including the alert severity (`kibana.alert.severity`) and the rule that generated the alert (`kibana.alert.rule.name`). Alert fields are interactive. Hover over alert fields to access multiple actions.
+
+* *Event renderer*: Surfaces important alert details to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.
```

Comment:
To illustrate the difference between the details in the _Alert-rendered reason statement_:

> **Alert-rendered reason statement**: Summarizes alert details

and the _Event renderer_ description:

> **Event renderer**: Surfaces important alert details

consider adding some examples to the Event renderer description, like the suggestion in _italics_ below:

**Event renderer**: Surfaces important alert details, _for example, file paths or process arguments_, to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.

Reply:
```suggestion
* *Event renderer*: Surfaces important alert details, for example, file paths or process arguments, to provide useful context for understanding the alert. Alert details are assembled into a human-readable format and alert fields are interactive.
```